crypto/dilithium/sig_dilithium3.c

```diff
@@ -1,40 +1,56 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
-// SPDX-License-Identifier: Apache-2.0
-
+// SPDX-License-Identifier: Apache-2.0 OR ISC
 #include "../evp_extra/internal.h"
 #include "../fipsmodule/evp/internal.h"
 #include "sig_dilithium.h"
-#include "pqcrystals_dilithium_ref_common/api.h"
+#include "pqcrystals_dilithium_ref_common/sign.h"
+#include "pqcrystals_dilithium_ref_common/params.h"
+
+#include "./pqcrystals_dilithium_ref_common/fips202.c"
+#include "./pqcrystals_dilithium_ref_common/ntt.c"
+#include "./pqcrystals_dilithium_ref_common/packing.c"
+#include "./pqcrystals_dilithium_ref_common/params.c"
+#include "./pqcrystals_dilithium_ref_common/poly.c"
+#include "./pqcrystals_dilithium_ref_common/polyvec.c"
+#include "./pqcrystals_dilithium_ref_common/reduce.c"
+#include "./pqcrystals_dilithium_ref_common/rounding.c"
+#include "./pqcrystals_dilithium_ref_common/sign.c"
+#include "./pqcrystals_dilithium_ref_common/symmetric-shake.c"
```

Comment:
Why include the C files like this? The only place we normally do this is in bcm.c.

Reply:
Added a comment in [c7e58db](https://github.com/aws/aws-lc/pull/1910/commits/c7e58db98ee2a6f93427cc66ea993b301e1590d2). We do this also for `ml_kem` (see https://github.com/aws/aws-lc/blob/main/crypto/fipsmodule/ml_kem/ml_kem.c#L9-L19) then include `ml_kem.c` in `bcm.c`.